src/util/ApiClient.ts

```diff
@@ -46,15 +48,26 @@ class ApiClient {
     }
 
     private sendRequest = async (method: RequestMethod, endpoint: endpoints, object: any) => {
+        const entityUrl = this.getEntityUrl(endpoint);
+        return this.sendRequestToUrl(method, entityUrl, object, 'include');
+    }
+
+    private getEntityUrl = (entityName: endpoints) => {
+        return `${API_URL}/${entityName}`;
+    }
+
+    public sendRequestToUrl = async (
```

Comment:
the difference in name between sendRequest and sendRequestToUrl is not declarative enough.

Reply:
I would suggest:
sendRequestToUrl => sendRequest
sendRequest => sendJoreMapRequest

Reply:
Fixed, its much better now